### PR TITLE
Add All Files

### PR DESCRIPTION
**Pull request by Balikuddembe**
In this milestone,i am implementing the code review technique on my own code. I am going to add comments on my code base so that i make it more professional and organized

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html lang="en">
```
Comment (Balikuddembe) [inline index.html:3]: Kindly I suggest that you remove empty lines in your code base 😸 This will make your code more readable.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  <title>Challenge</title>
+  <link rel="stylesheet" href="styles.css" </head>
+  <script src="challenge.js"></script>
+
+<body>
+  <h1>Adam Smith</h1>
+  <p id="bio">Lorem ipsum dolor sit amet consectetur adipisicing elit. Dignissimos <a
+      href="https://www.linkedin.com">linkedin </a>ipsum, ducimus saepe ratione repudiandae, rem ad doloribus quas
+    quidem alias dicta doloremque mollitia! Explicabo vel, earum vero velit nobis cupiditate.</p>
+  <h2>Education history</h2>
```
Comment (Balikuddembe) [inline index.html:18]: I request you to add this block of code in a div so that you have more control on the position of your project 😄

**Review by Balikuddembe — COMMENTED** [review]
# Changes required 💯 
You have done a great job so far.Just a few changes required before merging your code pull request.
Please check my comments below and make all required changes.
Thanks 😄

**Review by frank1738 — APPROVED** [review]
Hi @Balikuddembe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Good use of Github inline comments ✅
- Gave a reason for each required change✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**